### ISSUE TITLE
automatically save breakpoints

### ISSUE THREAD

**Issue by tromey** [tromey/gdb-gui#37]
nemiver automatically saves breakpoints per-executable.
This sounds like a handy feature to copy.

**Comment by ahmednofal** [comment]
yes please that is a very good feature, debugging non x86 via a server and a socket resets the execution a lot which renders the gdbgui web page non responsive and disconnects it from the server. The breakpoints are always removed.
